### ISSUE TITLE
null should be a valid type

### ISSUE THREAD

**Issue by jkufver** [rundfunk47/Juice#4]
`{"a": "Apple 0", "b": null}`

I get _Juice.NotDecodableJSONError_ when trying to convert the string as follows:

```
let string = "{\"a\": \"Apple\", \"b\": null}"
let dict = try! toStrictlyTypedJSON(toLooselyTypedJSON(string)) as! JSONDictionary
```

A have verified the json with https://jsonformatter.curiousconcept.com/ and `null` is a valid type.

**Comment by rundfunk47** [comment]
Yes, I've been meaning to fix this for a while - indeed, null is a valid value according to the JSON-specification. Fixed in version 1.0.10!

**Comment by jkufver** [comment]
Nice work Narek. Thanks a lot.
